docs/monitoring.md

```diff
@@ -473,6 +473,10 @@ can be identified by their `[attempt-id]`. In the API listed below, when running
     <td>
       A list of all stages for a given application.
       <br><code>?status=[active|complete|pending|failed]</code> list only stages in the state.
+      <br><code>?details=true</code> lists stages with the task data.
```

Comment:
Just checking, these docs are now up to date given the PRs that have been merged?

Reply:
> Just checking, these docs are now up to date given the PRs that have been merged?

Done